sklearn/linear_model/_coordinate_descent.py

```diff
@@ -224,14 +224,29 @@ def lasso_path(
 
         (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
 
+    .. math::
+        \\frac{1}{2n_{\\operatorname{samples}}}
+        \\vert \\vert y - Xw \\vert \\vert^2_2 +
```

Comment:
Isn't there a less verbose version of the Euclidean norm that is understood by sphinx?

Reply:
Alternatively, we could just use `||` no? That might make such expression a lot more readable when reading the docstring in an IDE (even if slightly less correctly from a typographical point of view).